openquake/ghm/mosaic.py

```diff
@@ -114,7 +114,7 @@
         'twn': ['TW'],
         'usa': ['US'],
         'ucf': ['UCF'],
-        'waf': ['AG', 'AO', 'BC', 'BN', 'CD', 'CF', 'CG', 'CM', 'CT', 'EK', 'GA', 
+        'waf': ['AG', 'BN', 'CD', 'CF', 'CM', 'CT', 'EK', 'GA', 
                 'GB', 'GH', 'GV', 'IV', 'LI', 'LY', 'ML', 'MR', 'NG', 'NI', 'PU',
-                'SG', 'SH', 'SL', 'SU', 'TO', 'TP', 'UV', 'WA'],
+                'SG', 'SH', 'SL', 'SU', 'TO', 'TP', 'UV'],
```

Comment:
this is the wrong version of mosaic.py - we do not want to remove these countries from WAF